### ISSUE TITLE
Uncaught TypeError: Cannot read property 'isVisible' of undefined

### ISSUE THREAD

**Issue by jcassanyjcs** [SqrTT/Bart#23]
[Enter steps to reproduce:]

1. Just tried to open my Bart settings this morning
2. And got that error. Doesn't allow me to use Bart

**Atom**: 1.16.0 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.4
**Thrown From**: [Bart](https://github.com/SqrTT/bart) package 1.6.0

### Stack Trace

Uncaught TypeError: Cannot read property 'isVisible' of undefined

```
At /Users/josecassany/.atom/packages/Bart/lib/bart.js:380

TypeError: Cannot read property 'isVisible' of undefined
    at /packages/Bart/lib/bart.js:380:30)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
    at CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:3:59)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/app.asar/src/command-registry.js:160:19)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchApplicationMenuCommand (/app.asar/src/atom-environment.js:1179:28)
    at EventEmitter.outerCallback (/app.asar/src/application-delegate.js:312:25)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -0:14.4.0 bart:openConfig (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.29.24 
atom-ternjs 0.18.3 
Bart 1.6.0 
busy-signal 1.4.1 
close-tags 0.6.0 
intentions 1.1.2 
language-demandware 0.1.0 
linter 2.1.4 
linter-ui-default 1.2.4 
linter-xo 0.23.0 
minimap 4.28.2 
```

**Comment by SqrTT** [comment]
thanks for reporting issue. Temporary as workaround you might use `dw.json` for access to configuration. I'll address this issue ASAP.

**Comment by jcassanyjcs** [comment]
I actually replaced the bart.js document for a previous one. I guess a some point there will be an update in the package right?
Thank you for your prompt response!

**Comment by SqrTT** [comment]
Correct. Update is available now. 

@jcassanyjcs Could you test and confirm that issue is addressed, please? Thanks.

**Comment by jcassanyjcs** [comment]
I opened a new Issue. Now I get a different error when I try to upload my cartridges :(

**Comment by SqrTT** [comment]
fixed according to #25
